.github/ISSUE_TEMPLATE/PAGE_FEEDBACK.yml

```diff
@@ -0,0 +1,27 @@
+name: Page Feedback
+description: >-
+  Used for page feedback from users visiting opentelemetry.io and clicking "no"
+  if a page was helpful.
+title: '[Page Feedback]: Your feedback'
```

Comment:
Is this the template title or the title that will be used when the template is instantiated? If the latter, can we replace "Your feedback" with "INSERT PAGE TITLE HERE" ... or just use "Page feedback". Or, add an optional field entitled "Page title".

Reply:
Sure.

Reply:
```suggestion
title: '[Page Feedback]: summary of your feedback'
```